CMakeLists.txt

```diff
@@ -230,8 +168,22 @@ add_executable(llvm2bpl
   tools/llvm2bpl/llvm2bpl.cpp
 )
 
+# We have to import LLVM's cmake definitions to build sea-dsa
+# Borrowed from sea-dsa's CMakeLists.txt
+find_package (LLVM 8.0 CONFIG NO_DEFAULT_PATH)
```

Comment:
Does this have to be updated whenever we update LLVM? Should this be `8.0` or `8.0.1`?

Reply:
I think so. It should be 8.0 or simply 8.

Reply:
The [version] argument requests a version with which the package found should be compatible (format is major[.minor[.patch[.tweak]]]). The EXACT option requests that the version be matched exactly.